server/settings/publish_plugins.py

```diff
@@ -380,6 +390,11 @@ class PublishPluginsModel(BaseSettingsModel):
         "primary_pool": "",
         "secondary_pool": ""
     },
+    "CollectAYONServerUrlToFarmJob": {
```

Comment:
Wouldn't be better to have settings for `CollectDeadlineJobEnvVars` with enum `Pass AYON server url to job` and options `Always, User choice, Never` instead of new plugin?